juju/docs
# Add documentation-builder to 2.2 docs

## Done
- Fixed a few things in the docs that brakes the documentation encoding to HTML
- Created a navigation metadata.yaml

## QA
- Check out branch 
- Run `documentation-builder --source-folder src --output-media-path 'static/media' --media-url '/static/media' --search-domain 'docs.jujucharms.com' --search-url 'https://docs.ubuntu.com/search' --search-placeholder 'Search Juju docs' --no-link-extensions`
- Open /build/en/ files
